### PR TITLE
added schema for USD transfers

### PR DESCRIPTION
**Pull request by SonalBhanawat**

### CODE REVIEW

File: fiatconnect-api.md [hunk 1]
```diff
@@ -2267,6 +2268,28 @@ The `ifsc` or Indian Financial System Code is an 11-digit alpha-numeric code tha
...
+`USDAccount` is an account schema that represents transfers for `USD` currency.
+
+```
+{
+	accountName: `string`,
+	institutionName: `string`,
+	accountNumber: `string`,
+	country: `string`,
+	fiatAccountType: `FiatAccountTypeEnum.BankAccount`,
+	branchIdentifier: `string`,
```
Comment (cajubelt) [inline fiatconnect-api.md:2282]: is this actually needed?

Comment (cajubelt) [reply fiatconnect-api.md:2282]: I've never had to provide it before for a transfer in the US. Wouldn't even know how to find mine

File: fiatconnect-api.md [hunk 2]
```diff
@@ -2267,6 +2268,28 @@ The `ifsc` or Indian Financial System Code is an 11-digit alpha-numeric code tha
 The `accountNumber` field is unique between individuals and no two banks or account holders can have the same account number.
 Banks use different starting codes for their branches for differentiation. (In India, bank account numbers usually contain 7 to 21 digits.)
 
+#### 8.3.2.6. `USDAccount`
```
Comment (cajubelt) [inline fiatconnect-api.md:2271]: probly needs to be added to table of contents

File: fiatconnect-api.md [hunk 3]
```diff
@@ -2110,7 +2110,8 @@ An enum listing the frequency, or how often, a particular fee needs to be paid.
 	`MobileMoney`,
 	`DuniaWallet`,
 	`IBANNumber`,
-	`IFSCAccount`
+	`IFSCAccount`,
+	`USDAccount`
```
Comment (cajubelt) [inline fiatconnect-api.md:2114]: can you update the swagger spec with this too?

**Review by cajubelt — COMMENTED** [review]
could a plaid token be used for this instead? would probably be better for security and user trust. we can discuss.

File: fiatconnect-api.md (hunk 1 above)
Comment (SonalBhanawat) [reply fiatconnect-api.md]: Removed branchIdentifier.

File: fiatconnect-api.md (hunk 2 above)
Comment (SonalBhanawat) [reply fiatconnect-api.md:2272]: done.

File: fiatconnect-api.md (hunk 3 above)
Comment (SonalBhanawat) [reply fiatconnect-api.md:2115]: done.

**Comment by cajubelt** [comment]
this is stale so I'm closing it, since my guess is that there are no immediate plans to use it. though it looks like it's in good shape, so if you wanted to reopen it and work on getting it merged that's fine too

**Pull request closed without merging**